<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>BioC2019</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


<link rel="stylesheet" href="my-theme.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">kevinrue's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Bioc2019.html">Bioc2019</a>
    </li>
    <li>
      <a href="Bioc2020.html">Bioc2020</a>
    </li>
    <li>
      <a href="EuroBioc2020.html">EuroBioc2020</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/kevinrue/twitter-stats">
    <span class="fas fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">BioC2019</h1>
<h3 class="subtitle">Bioconductor 2019 conference</h3>
<h4 class="date">2020-12-29 10:34:12</h4>

</div>


<p><strong>Parameters</strong></p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Parameter
</th>
<th style="text-align:left;">
Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hashtag
</td>
<td style="text-align:left;">
#BioC2019
</td>
</tr>
<tr>
<td style="text-align:left;">
start_day
</td>
<td style="text-align:left;">
2019-06-24
</td>
</tr>
<tr>
<td style="text-align:left;">
end_day
</td>
<td style="text-align:left;">
2019-06-27
</td>
</tr>
<tr>
<td style="text-align:left;">
timezone
</td>
<td style="text-align:left;">
America/New_York
</td>
</tr>
<tr>
<td style="text-align:left;">
theme
</td>
<td style="text-align:left;">
theme_light
</td>
</tr>
<tr>
<td style="text-align:left;">
accent
</td>
<td style="text-align:left;">
#0092ac
</td>
</tr>
<tr>
<td style="text-align:left;">
accent2
</td>
<td style="text-align:left;">
#87b13f
</td>
</tr>
<tr>
<td style="text-align:left;">
kcore
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
topics_k
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
bigram_filter
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
fixed
</td>
<td style="text-align:left;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
seed
</td>
<td style="text-align:left;">
1
</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the #BioC2019 hashtag for the <a href="http://bioc2019.bioconductor.org/" title="BioC 2019">Bioconductor 2019 conference</a>, 24-27 June 2019 online (originally planned in Boston, USA).</p>
<p>A total of 1563 tweets from 357 users were collected using the <code>rtweet</code> R package.</p>
<p>The latest tweet was recorded at 2019-06-28 14:27:11 (Etc/UTC).</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/Bioc2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-06-24 - 2019-06-27 in the America/New_York timezone.</p>
<p><img src="figures/Bioc2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered">
<h3>Overall</h3>
<p><img src="figures/Bioc2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered">
<h3>Original</h3>
<p><img src="figures/Bioc2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered">
<h3>Retweets</h3>
<p><img src="figures/Bioc2019/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.2</span> Retweet proportion</h2>
<p><img src="figures/Bioc2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2">
<h2><span class="header-section-number">3.3</span> Top tweeters timeline</h2>
<p><img src="figures/Bioc2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">3.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Overall</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/Bioc2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/Bioc2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/Bioc2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/Bioc2019/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Original</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/Bioc2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/Bioc2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/Bioc2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/Bioc2019/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade">
<h3>Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/Bioc2019/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/Bioc2019/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-2" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/Bioc2019/tweets-top-users-retweets-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-2" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/Bioc2019/tweets-top-users-retweets-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1">
<h1><span class="header-section-number">4</span> Sources</h1>
<p><img src="figures/Bioc2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">5</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">5.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/Bioc2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">5.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/Bioc2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1">
<h1><span class="header-section-number">6</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/Bioc2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/Bioc2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered">
<h3>Top 10</h3>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
The Bioconductor 2019 conference workshop materials are available online: <a href="https://t.co/kpdfKcgjZv" class="uri">https://t.co/kpdfKcgjZv</a>. Instructions for use locally are here: <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> #BioC2019 <a href="https://t.co/xhuuancrl1" class="uri">https://t.co/xhuuancrl1</a>
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
Just completed (hopefully) final version of the #bioc2019 workshop materials; 26 workshop packages with 607 dependent packages. All available via #docker image (and volume) and Amazon Machine Image. <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> Team of folks involved, not to mention workshop authors.
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
<p>Interested in clustering large #singlecell #rnaseq data with #rstats in a fast and scalable way with data in memory or on-disk (e.g. #hdf5)? Check out my talk in the next section after the break! #BioC2019</p>
<p>📣slides: <a href="https://t.co/baaP1dQLIA" class="uri">https://t.co/baaP1dQLIA</a></p>
💻software: <a href="https://t.co/0eBOapo6SJ" class="uri">https://t.co/0eBOapo6SJ</a>
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Martin Morgan telling us that <span class="citation">@Bioconductor</span> has 1751 packages, 1200 maintainers, and was downloaded by over a half million unique IPs last year. Hello from #bioc2019! <a href="https://t.co/pD4D4CuJb5" class="uri">https://t.co/pD4D4CuJb5</a>
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
KasperDHansen
</td>
<td style="text-align:left;">
My lab is hiring post docs (and graduate students). We have multiple collaborators and former lab members at #bioc2019 (unfortunately, I couldn’t attend this year).
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
In case it’s useful, here are my notes from Developer Day at #BioC2019 <a href="https://t.co/vkJKiWaT6G" class="uri">https://t.co/vkJKiWaT6G</a> (inspired by <span class="citation">@jxtx</span> last year) I’ll try to continue over the next few days and write something up following the meeting
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Robert Gentleman: “If people can’t do things that piss you off then you haven’t given them any power.” (in reference to opening up early #rstats development beyond he and Ross Ihaka and the beginnings of R packages) #BioC2019
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
tangming2005
</td>
<td style="text-align:left;">
single cell people, you really should try iSEE <a href="https://t.co/BHncVXsNLM" class="uri">https://t.co/BHncVXsNLM</a> awesome package #bioc2019
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
AK: introducing Kipoi <a href="https://t.co/mUW5n46gHi" class="uri">https://t.co/mUW5n46gHi</a> <a href="https://t.co/BwQl69U6q5" class="uri">https://t.co/BwQl69U6q5</a> to facilitate the exchange and reuse of predictive models for genomics #bioc2019 <a href="https://t.co/eehcE5gP9l" class="uri">https://t.co/eehcE5gP9l</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
Build an academic website in 10 minutes.. Got no excuse now!! #bioc2019 <span class="citation">@bioconductor</span> from <span class="citation">@KevinRUE67</span> #TODOLIST #academiclife <a href="https://t.co/I9wcUi8UV1" class="uri">https://t.co/I9wcUi8UV1</a> <a href="https://t.co/75zHdN4eCl" class="uri">https://t.co/75zHdN4eCl</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered">
<h3>Most retweeted</h3>
<p><img src="figures/Bioc2019/most-retweeted-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/Bioc2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/Bioc2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered">
<h3>Top 10</h3>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
Just completed (hopefully) final version of the #bioc2019 workshop materials; 26 workshop packages with 607 dependent packages. All available via #docker image (and volume) and Amazon Machine Image. <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> Team of folks involved, not to mention workshop authors.
</td>
<td style="text-align:right;">
96
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Martin Morgan telling us that <span class="citation">@Bioconductor</span> has 1751 packages, 1200 maintainers, and was downloaded by over a half million unique IPs last year. Hello from #bioc2019! <a href="https://t.co/pD4D4CuJb5" class="uri">https://t.co/pD4D4CuJb5</a>
</td>
<td style="text-align:right;">
91
</td>
</tr>
<tr>
<td style="text-align:left;">
tangming2005
</td>
<td style="text-align:left;">
Presenting a poster at #bioc2019 <a href="https://t.co/xMxPmENBsX" class="uri">https://t.co/xMxPmENBsX</a>
</td>
<td style="text-align:right;">
87
</td>
</tr>
<tr>
<td style="text-align:left;">
e_Karaesmen
</td>
<td style="text-align:left;">
Awesome to see many #rladies at #BioC2019 maybe we should start #BioCLadies too 😜 <a href="https://t.co/vxldiqf6iv" class="uri">https://t.co/vxldiqf6iv</a>
</td>
<td style="text-align:right;">
84
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
The Bioconductor 2019 conference workshop materials are available online: <a href="https://t.co/kpdfKcgjZv" class="uri">https://t.co/kpdfKcgjZv</a>. Instructions for use locally are here: <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> #BioC2019 <a href="https://t.co/xhuuancrl1" class="uri">https://t.co/xhuuancrl1</a>
</td>
<td style="text-align:right;">
67
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
In case it’s useful, here are my notes from Developer Day at #BioC2019 <a href="https://t.co/vkJKiWaT6G" class="uri">https://t.co/vkJKiWaT6G</a> (inspired by <span class="citation">@jxtx</span> last year) I’ll try to continue over the next few days and write something up following the meeting
</td>
<td style="text-align:right;">
64
</td>
</tr>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
<p>Interested in clustering large #singlecell #rnaseq data with #rstats in a fast and scalable way with data in memory or on-disk (e.g. #hdf5)? Check out my talk in the next section after the break! #BioC2019</p>
<p>📣slides: <a href="https://t.co/baaP1dQLIA" class="uri">https://t.co/baaP1dQLIA</a></p>
💻software: <a href="https://t.co/0eBOapo6SJ" class="uri">https://t.co/0eBOapo6SJ</a>
</td>
<td style="text-align:right;">
62
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
Ready for the <span class="citation">@Bioconductor</span> meeting #bioc2019 in NYC, bringing along stickers for the two most popular packages I maintain. Reach me out if you want one!! <a href="https://t.co/xo5iDqduLo" class="uri">https://t.co/xo5iDqduLo</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
In my cancer journal entry today I tried to express how happy I am to be at the #bioc2019 <span class="citation">@Bioconductor</span> conference. <a href="https://t.co/6ykJw06qFy" class="uri">https://t.co/6ykJw06qFy</a>
</td>
<td style="text-align:right;">
59
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
On route to #bioc2019 wearing my comfy #user2018 socks. Although my head isn’t quite there yet (jetlag!) my feet are fully prepared :) Presenting 2 workshops on #RNAseq analysis this Wed for those interested <a href="https://t.co/htVrPanrqz" class="uri">https://t.co/htVrPanrqz</a>
</td>
<td style="text-align:right;">
57
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered">
<h3>Most likes</h3>
<p><img src="figures/Bioc2019/most-favorited-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills">
<h2><span class="header-section-number">6.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered">
<h3>Proportion</h3>
<p><img src="figures/Bioc2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered">
<h3>Count</h3>
<p><img src="figures/Bioc2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered">
<h3>Top 10</h3>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
RevDocGabriel
</td>
<td style="text-align:left;">
Thanks <span class="citation">@LeviWaldron1</span> for taking the #BioC2019 crew on a tour of #CentralPark #Trailblazer <a href="https://t.co/qnB2rtpidU" class="uri">https://t.co/qnB2rtpidU</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
josschavezf1
</td>
<td style="text-align:left;">
Such a great experience today! I met awesome people at #BioC2019 <a href="https://t.co/sMkXCNTBQZ" class="uri">https://t.co/sMkXCNTBQZ</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
danicassol
</td>
<td style="text-align:left;">
It’s a privilege to be part of this amazing community #bioc2019 #bioconductor <a href="https://t.co/OFIKL5XzAr" class="uri">https://t.co/OFIKL5XzAr</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
amorim_cfa
</td>
<td style="text-align:left;">
As a biologist, the world of bioinformatics is attracting more and more - that’s what I want to do. Just came back from #BioC2019, excellent experience, talks, networking… it was an amazing conference! #Rladies #NewYork #bioinformatics <a href="https://t.co/Z7wX5t2l9t" class="uri">https://t.co/Z7wX5t2l9t</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
e_Karaesmen
</td>
<td style="text-align:left;">
<span class="citation">@JeanneGarb</span> Oh I think you missed us because we were taking a photo… As a female PhD student I think <span class="citation">@Bioconductor</span> is one the most inclusive tech communities in the world. #BioC2019 <a href="https://t.co/ft1u7XT4a0" class="uri">https://t.co/ft1u7XT4a0</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
And great to have spread the word to new ones who had never heard about #rladies <span class="citation">@RLadiesGlobal</span> #BioC2019 <a href="https://t.co/cHbjuvkxJW" class="uri">https://t.co/cHbjuvkxJW</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
This is one conference I wish to attend in future. #BioC2019 seem 🔥. Twitter is all I have to learn about it this year. Bring it on. Something I learnt early on. Don’t reinvent wheel if you Jane options, one can work their way through if one digs deep. # <span class="citation">@Bioconductor</span> speaks! <a href="https://t.co/sabVIm0cyC" class="uri">https://t.co/sabVIm0cyC</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
claudiawultsch
</td>
<td style="text-align:left;">
Excited to be joining in tomorrow #BioC2019 #Bioconductor <a href="https://t.co/YCQZa2aRfs" class="uri">https://t.co/YCQZa2aRfs</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Argh, after the break we have parallel sessions! Anyone attending the ‘Statistical methods’ section want to take notes? #Bioc2019 <a href="https://t.co/9XKcFo6DZU" class="uri">https://t.co/9XKcFo6DZU</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
UofABioinfoHub
</td>
<td style="text-align:left;">
Here’s the URL from Pete’s notes in case it gets hidden on a retweet: <a href="https://t.co/RAqXSuKFVj" class="uri">https://t.co/RAqXSuKFVj</a> #BioC2019 <a href="https://t.co/YQibd6RnpV" class="uri">https://t.co/YQibd6RnpV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered">
<h3>Most quoted</h3>
<p><img src="figures/Bioc2019/most-quoted-1.png" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills">
<h1><span class="header-section-number">7</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered">
<h2>Proportion</h2>
<p><img src="figures/Bioc2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered">
<h2>Top 10</h2>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Martin Morgan telling us that <span class="citation">@Bioconductor</span> has 1751 packages, 1200 maintainers, and was downloaded by over a half million unique IPs last year. Hello from #bioc2019! <a href="https://t.co/pD4D4CuJb5" class="uri">https://t.co/pD4D4CuJb5</a>
</td>
<td style="text-align:right;">
91
</td>
</tr>
<tr>
<td style="text-align:left;">
tangming2005
</td>
<td style="text-align:left;">
Presenting a poster at #bioc2019 <a href="https://t.co/xMxPmENBsX" class="uri">https://t.co/xMxPmENBsX</a>
</td>
<td style="text-align:right;">
87
</td>
</tr>
<tr>
<td style="text-align:left;">
e_Karaesmen
</td>
<td style="text-align:left;">
Awesome to see many #rladies at #BioC2019 maybe we should start #BioCLadies too 😜 <a href="https://t.co/vxldiqf6iv" class="uri">https://t.co/vxldiqf6iv</a>
</td>
<td style="text-align:right;">
84
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
The Bioconductor 2019 conference workshop materials are available online: <a href="https://t.co/kpdfKcgjZv" class="uri">https://t.co/kpdfKcgjZv</a>. Instructions for use locally are here: <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> #BioC2019 <a href="https://t.co/xhuuancrl1" class="uri">https://t.co/xhuuancrl1</a>
</td>
<td style="text-align:right;">
67
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
Ready for the <span class="citation">@Bioconductor</span> meeting #bioc2019 in NYC, bringing along stickers for the two most popular packages I maintain. Reach me out if you want one!! <a href="https://t.co/xo5iDqduLo" class="uri">https://t.co/xo5iDqduLo</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
On route to #bioc2019 wearing my comfy #user2018 socks. Although my head isn’t quite there yet (jetlag!) my feet are fully prepared :) Presenting 2 workshops on #RNAseq analysis this Wed for those interested <a href="https://t.co/htVrPanrqz" class="uri">https://t.co/htVrPanrqz</a>
</td>
<td style="text-align:right;">
57
</td>
</tr>
<tr>
<td style="text-align:left;">
QianLiu28878838
</td>
<td style="text-align:left;">
Bioconductor team is celebrating the success of 15th conference in NYC! #BioC2019 <a href="https://t.co/2MlfXxkZji" class="uri">https://t.co/2MlfXxkZji</a>
</td>
<td style="text-align:right;">
51
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
⁦<span class="citation">@Bioconductor</span>⁩ #bioc2019 in #NYC <a href="https://t.co/m5qbaQpXd6" class="uri">https://t.co/m5qbaQpXd6</a>
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
michellemiron3
</td>
<td style="text-align:left;">
Avi presents a tool he created for estimating gene abundance from single-cell RNA seq data #bioc2019 <a href="https://t.co/nXBr2dNwCW" class="uri">https://t.co/nXBr2dNwCW</a>
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
New Waldron Lab photo at #bioc2019! I couldn’t be prouder of a group that works so happily and productively together, or of the four besides me who presented in workshops this year. We contributed: (1/2) <span class="citation">@cmirzayi</span> <span class="citation">@drsehyun</span> <span class="citation">@M2RUseR</span> &amp;amp; alumnus <span class="citation">@_schifferl</span>. <a href="https://t.co/6EwYcadr2L" class="uri">https://t.co/6EwYcadr2L</a> <a href="https://t.co/oCFZipEPCy" class="uri">https://t.co/oCFZipEPCy</a>
</td>
<td style="text-align:right;">
40
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore">
<h1></h1>
<div id="most-liked-image" class="section level2">
<h2><span class="header-section-number">7.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/D91FlYFWwAAH2p9.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">8</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">8.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/Bioc2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">8.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/Bioc2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">8.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/Bioc2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade">
<h3><span class="header-section-number">8.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PapaemmanuilLab
</td>
<td style="text-align:left;">
Real pleasure to introduce #Isabl at #bioc2019 this am. Very gratifying to see the response! Structured data architectures as digital biobanks, secure data governance, reproducibility and foster data sharing, analyses and innovation. Working towards Isabl release soon #staytuned <a href="https://t.co/sPMGfToWTk" class="uri">https://t.co/sPMGfToWTk</a>
</td>
<td style="text-align:right;">
0.9946292
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@anshulkundaje</span> AK is modeling transcription factor data from ChIP-seq data or chromatin accessibility data from DNAase-seq or ATAC-seq. How do we predict if individual sites are active or inactive in terms of transcription binding? #BioC2019
</td>
<td style="text-align:right;">
0.9943973
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Nathan Sheffield presents a well-engineered solution to data provenance in the metadata slot of <span class="citation">@Bioconductor</span> objects. You use a single yaml file containing locations of your source data file + annotations, and R script to load data <a href="https://t.co/IQzBPUwhiY" class="uri">https://t.co/IQzBPUwhiY</a> #BioC2019 <a href="https://t.co/Bh8PSyq8hf" class="uri">https://t.co/Bh8PSyq8hf</a>
</td>
<td style="text-align:right;">
0.9941444
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
SA. New seq alignment which is halfway between salmon quasi and full alignment approach. Towards selective-alignment: Bridging the accuracy gap between alignment-based and alignment-free transcript quantification | bioRxiv #bioc2019 <a href="https://t.co/wwfoCZoOO8" class="uri">https://t.co/wwfoCZoOO8</a>
</td>
<td style="text-align:right;">
0.9938676
</td>
</tr>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
<p>Interested in clustering large #singlecell #rnaseq data with #rstats in a fast and scalable way with data in memory or on-disk (e.g. #hdf5)? Check out my talk in the next section after the break! #BioC2019</p>
<p>📣slides: <a href="https://t.co/baaP1dQLIA" class="uri">https://t.co/baaP1dQLIA</a></p>
💻software: <a href="https://t.co/0eBOapo6SJ" class="uri">https://t.co/0eBOapo6SJ</a>
</td>
<td style="text-align:right;">
0.9935634
</td>
</tr>
<tr>
<td style="text-align:left;">
k3yavi
</td>
<td style="text-align:left;">
#bioc2019 the effect on quantification estimates based on the different alignment/mapping methods used on “real” data, compared to hybrid Oracle . The problem of RNA-seq quantification is not solved… <a href="https://t.co/qtsVMhj9aa" class="uri">https://t.co/qtsVMhj9aa</a>
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Key slides from Nitesh Turaga’s presentation on Docker for easy dev, use, and for creating clusters on the cloud called from R session. Slides at <a href="https://t.co/Go7KW4uChu" class="uri">https://t.co/Go7KW4uChu</a> #BioC2019 #Docker <a href="https://t.co/pwlQmhzFBQ" class="uri">https://t.co/pwlQmhzFBQ</a>
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@PapaemmanuilLab</span> EP describes ISABL, a platform for scalable biostat informatics analysis - includes RESTful API, integrated with institutional RedCap, can run everything on laptop in 10 minutes #bioc2019
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
CS: applied different quantification methods to nano-ore and illumina data and both technologies provide comparable correlations between replicates. So, yes, nanopore data is quantitative #bioc2019
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
At the Robert Gentleman’s 60th Birthday Symposium, Richard Bourgon from <span class="citation">@genentech</span> gives a definition of what is a computational biologist: “computational is a modifier, we’re biological scientists” #bioc2019 <a href="https://t.co/zSINCGFWI5" class="uri">https://t.co/zSINCGFWI5</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Martin Morgan <span class="citation">@mt_morgan</span> (<span class="citation">@Bioconductor</span> lead) at the Robert Gentleman symposium</p>
<p>“We are still using S4” /jk</p>
<p>Last yr 500k unique IP for downloads 30k PubMed citations (vs ~1.4k when MM took over)</p>
<p>RG gave him the keys to THE car, not a Ferrari, but still the car! 😱</p>
#BioC2019
</td>
<td style="text-align:right;">
0.9946292
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@AedinCulhane</span> AC: moGSA is a PCA-based method and can also be use for single sample GSEA (ssGSEA). She applied it to finding PanCancer immune subtypes in TCGA data by generating ssGSEA scores for immune sets and pathways and then clustering them. #BioC2019
</td>
<td style="text-align:right;">
0.9943973
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
Pete <span class="citation">@PeteHaitch</span> is teaching us about the #DelayedArray framework developed by Hervé Pàges from <span class="citation">@Bioconductor</span> and now is showing us his #DelayedMatrixStats 📦 which enables matrix statistical functions on HDF5 data matrices. Motivated by #scRNAseq and #DNAm #WGBS data #BioC2019 <a href="https://t.co/FG8jTtNQvI" class="uri">https://t.co/FG8jTtNQvI</a>
</td>
<td style="text-align:right;">
0.9938676
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
To save time, I’ve reformatted my lightning talk as a tweet! If you’re a package developer, consider creating a #hexsticker and add it to the growing collection at <a href="https://t.co/mrwPKBnhMR" class="uri">https://t.co/mrwPKBnhMR</a>. For some cool examples see 👇 #bioc2019 <a href="https://t.co/VMXc4xeOud" class="uri">https://t.co/VMXc4xeOud</a>
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
RevDocGabriel
</td>
<td style="text-align:left;">
Such an honour for two avid learnRs from <span class="citation">@umiamimedicine</span> to meet the grandmaster of R himself, Prof. Robert Gentleman, at #BioC2019. Thank you, <span class="citation">@Bioconductor</span>, for this wonderful opportunity and the awards to travel to <span class="citation">@RockefellerUniv</span>. We will be back next year! #Rstats #R4DS <a href="https://t.co/7R9axo7dBS" class="uri">https://t.co/7R9axo7dBS</a>
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
anshulkundaje
</td>
<td style="text-align:left;">
#bioc2019 community - we’d love to have u involved in contributing to, developing, using, providing feedback about <span class="citation">@KipoiZoo</span> - ML model zoo for genomics. We’re inspired by the amazing <span class="citation">@Bioconductor</span> community. <a href="https://t.co/qMvaxZ1Xib" class="uri">https://t.co/qMvaxZ1Xib</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
areyesq
</td>
<td style="text-align:left;">
Charlotte <span class="citation">@CSoneson</span> is telling us about TreeSummarizedExperiment. That extends SE by adding tree structures to rows and columns of the object. Looks useful for evolution, single cell analyses, etc. #BioC2019 <a href="https://t.co/NOSCnPLs61" class="uri">https://t.co/NOSCnPLs61</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Aedin Culhane <span class="citation">@AedinCulhane</span> now presenting on discovery of cancer immune molecular subtypes. She describes the moGSA package for multi-omics gene-set analysis <a href="https://t.co/FdKPSImLOX" class="uri">https://t.co/FdKPSImLOX</a> #BioC2019
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Denise Scholtens at the Robert Gentleman:</p>
<p>“Have you heard about reproducible research? It takes more time, but in the long run, it’s a lot easier for everyone”</p>
#BioC2019 #rstats #reproducibility <a href="https://t.co/tonEZHEKAL" class="uri">https://t.co/tonEZHEKAL</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
Adding up the framework model i.e CNN based to give an understanding of underlying NN layers of input, hidden and output layers. #bioc2019 <a href="https://t.co/Acf6Waabto" class="uri">https://t.co/Acf6Waabto</a>
</td>
<td style="text-align:right;">
0.9914399
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
<span class="citation">@cmirzayi</span> I’m <em>so</em> sorry to hear that Chloe. I do hope you report it to #bioc2019 committee. I know this doesn’t nes help but I can def relate. I’ve faced sim lack/conf comments my entire life &amp;amp; still do today. I’m happy to talk anytime about my exps &amp;amp; how to move past in acad. Just DM me!
</td>
<td style="text-align:right;">
0.9952226
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Wow! I didn’t know this! It’s mind blowing 🤯 Thanks for sharing this <span class="citation">@AedinCulhane</span>!</p>
<p>Men take 1 yr paternity leave/extension, get more publications. Women take 1 yr, don’t get as many pubs. Hence this mat/pat extension policy hurts on avg more women!</p>
#diversity #BioC2019 <a href="https://t.co/Svz5UCayj5" class="uri">https://t.co/Svz5UCayj5</a>
</td>
<td style="text-align:right;">
0.9946292
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Robert Castelo <span class="citation">@robertclab</span> is looking at RNAseq and proteomics from neonatal dried blood spots (DBS). He’s focusing on preterm infants, especially extremely low gestational age newborns (ELGANs) from whom it would be very challenging to draw peripheral blood. #BioC2019
</td>
<td style="text-align:right;">
0.9941444
</td>
</tr>
<tr>
<td style="text-align:left;">
k3yavi
</td>
<td style="text-align:left;">
Just arrived at NYC for #BioC2019 . Psyched for 3 days of awesome learning, cheers to <span class="citation">@Bioconductor</span> travel fellowship for giving the opportunity. For single cell quantification enthusiasts, come checkout alevin poster in Weiss lobby from 5:30 to 7:30pm on Tuesday.
</td>
<td style="text-align:right;">
0.9941444
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Aedin <span class="citation">@AedinCulhane</span> showed the history of women &amp;amp; #diversity in biostatistics &amp;amp; in the <span class="citation">@Bioconductor</span> project</p>
<p>Wording, quizzes (so women realize they are indeed qualified to attend) &amp;amp; most significantly direct outreach ✅ helped drive up women apps to <span class="citation">@ncbi</span>’s hackathon</p>
#BioC2019
</td>
<td style="text-align:right;">
0.9938676
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@lievenClement</span> LC said that, despite having really good properties, MsqRob has been difficult to disseminate given the lack of protein summaries and the fact that a mixed model approach only allows for asymptotic inference, without being to calculate degrees of freedom for t test. #BioC2019
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
I want to state that the behavior described by <span class="citation">@cmirzayi</span> after her workshop likely violated the #BioC2019 CoC rules against intellectual bullying. Such behavior reported anonymously or non-anonymously during the conference will be addressed promptly <a href="https://t.co/XyD4SgbxJf" class="uri">https://t.co/XyD4SgbxJf</a> (1/2) <a href="https://t.co/VLgwEzebOi" class="uri">https://t.co/VLgwEzebOi</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
KevinRUE67
</td>
<td style="text-align:left;">
Fellow #bioc2019 attendees: if you’re staying a few extra days, an awesome way of moving through the city is the 7-day metro card ($33). One trip is $2.75, in comparison. <a href="https://t.co/g78Km9g0z7" class="uri">https://t.co/g78Km9g0z7</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
PayalYokota
</td>
<td style="text-align:left;">
Great day at the #bioc2019 <span class="citation">@RockefellerUniv</span> today. Saw some #scRNAseq and bumped into some conf posse. Pumped about the 4pm workshops. But, first.. taking a breather to enjoy a delicious mocha <span class="citation">@lepainquotidien</span>. <a href="https://t.co/w8kBqmTUHi" class="uri">https://t.co/w8kBqmTUHi</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
mattbrauer
</td>
<td style="text-align:left;">
In NYC for #BioC2019 and went to see #Stonewall yesterday–almost 50 years to the day after the resistance was launched. I was surprised at how moving it was, but after all it represents a major turning point in human history. People’s minds can change, though it may take 50 yrs.
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CDSBMexico
</td>
<td style="text-align:left;">
<p>If you are interested in joining our Slack space please do so at <a href="https://t.co/lOCIVqWzse" class="uri">https://t.co/lOCIVqWzse</a></p>
<p>¿Te interesa ser parte de nuestra comunidad? Únete vía <a href="https://t.co/lOCIVqWzse" class="uri">https://t.co/lOCIVqWzse</a></p>
<p>FYI most channels are currently in Spanish, but we welcome English-speaking users</p>
#BioC2019 #rstats #rstatsES
</td>
<td style="text-align:right;">
0.9941444
</td>
</tr>
<tr>
<td style="text-align:left;">
cmirzayi
</td>
<td style="text-align:left;">
#Bioc2019 Hype post for my workshop this afternoon: Epidemiology for Bioinformaticians. I promise you dogs, DAGs, cats, confounders, an ambulance, toys, GWAS, MR, and a small amount of sleep disruption! Today at 4:10 in Weiss 305. Here’s the cat! <a href="https://t.co/KdG7PTg9vK" class="uri">https://t.co/KdG7PTg9vK</a>
</td>
<td style="text-align:right;">
0.9935634
</td>
</tr>
<tr>
<td style="text-align:left;">
areyesq
</td>
<td style="text-align:left;">
Ana’s great talk about the integration of COPD and IPF molecular profiling data for easy access for clinicians and wet lab researchers (as a web resource) and bioinformaticians as an R package. #BioC2019 <a href="https://t.co/KW0CPVjEPm" class="uri">https://t.co/KW0CPVjEPm</a>
</td>
<td style="text-align:right;">
0.9932273
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
JL: used known sex phenotype in GTEx &amp;amp; TCGA to predict that missing phenotype in recount2 <a href="https://t.co/Ex0eVN2sVz" class="uri">https://t.co/Ex0eVN2sVz</a> #BioC2019 now describing the broader problem of predicting genotypes from gene expression data
</td>
<td style="text-align:right;">
0.9924378
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>It’s great to see others re-using our work =)</p>
<p>See TCGAbiolinks::TCGAquery_recount2()</p>
<p>Congrats 🎉 #TCGAbiolinks team on <a href="https://t.co/DJyn6wNJHf" class="uri">https://t.co/DJyn6wNJHf</a> ^^</p>
<p>✅ Cites #recount2, #RailRNA and the recount workflow (a version of which I’ll teach at #BioC2019 next week)</p>
#rstats <span class="citation">@Bioconductor</span> <a href="https://t.co/Fy5MaXGhWy" class="uri">https://t.co/Fy5MaXGhWy</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Really nice setup for the workshops at #bioc2019! Can use Amazon Machine Instances (AMIs) for <span class="citation">@rstudio</span> - seems to have minimal lag! Also have the tutorials that folks can follow with/copy paste from at <a href="https://t.co/LAwz8IdvnL" class="uri">https://t.co/LAwz8IdvnL</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@acolaprico</span> AC also presented DeepBlueR <a href="https://t.co/3L0uUR1Ix1" class="uri">https://t.co/3L0uUR1Ix1</a>, for accessing epigenetics data, and MoonlightR <a href="https://t.co/nralZMAMqd" class="uri">https://t.co/nralZMAMqd</a> for identifying cancer driver genes (oncogenes and tumor suppressors) #BioC2019
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p>Day 2: <span class="citation">@jtleek</span> using recount2 for prediction of missing phenotype using known sex phenotype in GTEx &amp;amp; TCGA App link: <a href="https://t.co/BGzPiMeBCR" class="uri">https://t.co/BGzPiMeBCR</a></p>
<p>#BioC2019</p>
Slide link: <a href="https://t.co/A3T9aKfIU1" class="uri">https://t.co/A3T9aKfIU1</a>
</td>
<td style="text-align:right;">
0.9914399
</td>
</tr>
<tr>
<td style="text-align:left;">
PayalYokota
</td>
<td style="text-align:left;">
Great session about QC’ing those #ATACseq libraries by Haibo and chime ins from Julie and Jianhong. Unfortunately, the R didn’t seem to load libraries for hands-on but I think we all have enough legos to build our own, at home. #BioC2019
</td>
<td style="text-align:right;">
0.9914399
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Wolfgang Huber <span class="citation">@wolfgangkhuber</span> at the Robert Gentleman symposium highlighted how the <span class="citation">@Bioconductor</span> project is centered around turning users into developers, biologists into statisticians, statisticians into biologists and much more!!</p>
#BioC2019 <a href="https://t.co/8hZEtJK2Nb" class="uri">https://t.co/8hZEtJK2Nb</a>
</td>
<td style="text-align:right;">
0.9914399
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dommajeur
</td>
<td style="text-align:left;">
#Canicule #chaleur #abeilles #biodiversite #biotech #BioC2019 #bio #biodiversity #ESPUSA #agri #agriculture #charliehebdo #mediapartlive #pollution #pollen #ClimateChangeIsReal #ClimateChange #franceinfo #franceinter #France5 #France #TF1 #Le20hLeMag <a href="https://t.co/pqNrlJuLyw" class="uri">https://t.co/pqNrlJuLyw</a>
</td>
<td style="text-align:right;">
0.9943973
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
Just completed (hopefully) final version of the #bioc2019 workshop materials; 26 workshop packages with 607 dependent packages. All available via #docker image (and volume) and Amazon Machine Image. <a href="https://t.co/59hjl9F36i" class="uri">https://t.co/59hjl9F36i</a> Team of folks involved, not to mention workshop authors.
</td>
<td style="text-align:right;">
0.9941444
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Copy number variation analysis with Bioconductor: Data representation and manipulation, Integrative downstream analysis (eQTL, GWAS, …), Allele-specific CN analysis in cancer. <span class="citation">@M2RUseR</span>, <span class="citation">@drsehyun</span>, Ludwig Geistlinger. #bioc2019 <a href="https://t.co/hVuk5ftFZH" class="uri">https://t.co/hVuk5ftFZH</a>
</td>
<td style="text-align:right;">
0.9935634
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Matt <span class="citation">@matthewnmccall</span>, <span class="citation">@jhubiostat</span> alumni, introducing current <span class="citation">@jhubiostat</span> faculty <span class="citation">@stephaniehicks</span> at #BioC2019 🙌🏽</p>
Co-founder of the <span class="citation">@CorrespondAuth</span> <span class="citation">@RLadiesBmore</span> and a force behind #scRNAseq methods &amp;amp; analysis development using <span class="citation">@Bioconductor</span> + documention writing w <span class="citation">@drisso1893</span> <a href="https://t.co/B0Jzen2ldJ" class="uri">https://t.co/B0Jzen2ldJ</a>
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
mikelove
</td>
<td style="text-align:left;">
<p>My #BioC2019 workshop is 4:10-5pm in Weiss 305. It’s right after <span class="citation">@_StuartLee</span>’s workshop at 3:10.</p>
<p>plyranges is a great Bioc pkg for range operations w/ dplyr-like syntax, developed by <span class="citation">@_StuartLee</span>, <span class="citation">@lawremi</span>, <span class="citation">@visnut</span></p>
I’ll show using plyranges with tximeta <a href="https://t.co/op2AZJdaF8" class="uri">https://t.co/op2AZJdaF8</a>
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
robamezquita
</td>
<td style="text-align:left;">
Happy to see that <span class="citation">@Bioconductor</span> recognizes that it needs to diversify it’s leadership/advisory boards. It’s tough to admit that out loud, but at #bioc2019 they took a huge positive step forward into the future. Can’t wait to see the new initiatives that will grow from this moment
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@jtleek</span> JL asking what makes primary cancer different from metastatic cancer. Want to make this easy for many people to explore, eg by using recount2 <em>but</em> this uses SRA data, which has mixed or poor quality meta-data. #bioc2019
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
This is one conference I wish to attend in future. #BioC2019 seem 🔥. Twitter is all I have to learn about it this year. Bring it on. Something I learnt early on. Don’t reinvent wheel if you Jane options, one can work their way through if one digs deep. # <span class="citation">@Bioconductor</span> speaks! <a href="https://t.co/sabVIm0cyC" class="uri">https://t.co/sabVIm0cyC</a>
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
robamezquita
</td>
<td style="text-align:left;">
<span class="citation">@RevDocGabriel</span> Big fan of yours thanks to your question - I see no reason why we should be exclusivist regarding creating artificial barriers to entry for new developers. Lots of great statistical talent that is untapped that could make great genomic analysis software #bioc2019
</td>
<td style="text-align:right;">
0.9919697
</td>
</tr>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
That 😊feeling when you are leaving the longest and coldest ❄ nights in Melbourne to go to summer☀in NY! Exciting week ahead attending #bioc2019 and visiting <span class="citation">@jhubiostat</span> for the first time! #rstatstour
</td>
<td style="text-align:right;">
0.9914399
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Took me a while, but I can now run</p>
<p>Install <a href="https://t.co/5s6anvxrFT" class="uri">https://t.co/5s6anvxrFT</a> docker login docker run hello-world</p>
<p>Next docker pull bioconductor/release_base2 docker run -p 8787:8787 -e PASSWORD=bioc<br />
&amp;gt; bioconductor/release_base2:latest</p>
Thx <span class="citation">@niteshturaga</span> <span class="citation">@Bioconductor</span> #rstats #BioC2019✅ <a href="https://t.co/h6w8nZEuaF" class="uri">https://t.co/h6w8nZEuaF</a>
</td>
<td style="text-align:right;">
0.9938676
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
Truly excellent #bioc2019. I was so impressed with the high quality badges designed by <span class="citation">@jotsetung</span> and ⁦<span class="citation">@_schifferl</span>⁩. The local organizers ⁩Tomas Carroll ⁦<span class="citation">@RockefellerUniv</span>⁩, Kelly Ruggles ⁦<span class="citation">@nyuniversity</span>⁩ and ⁦<span class="citation">@LeviWaldron1</span>⁩ went the extra mile P&amp;lt;0.05 <a href="https://t.co/JIegSGHZsr" class="uri">https://t.co/JIegSGHZsr</a>
</td>
<td style="text-align:right;">
0.9938676
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Benjamin Haibe-Kains <span class="citation">@bhaibeka</span> now presenting on robust biomarker discovery from cancer pharmacogenomic data. Can use his group’s packages <a href="https://t.co/o5dqAR8nbZ" class="uri">https://t.co/o5dqAR8nbZ</a> to get datasets and analyze the data. He’s focused on robust biomarkers that are not platform-dependent #BioC2019
</td>
<td style="text-align:right;">
0.9935634
</td>
</tr>
<tr>
<td style="text-align:left;">
CDSBMexico
</td>
<td style="text-align:left;">
<p>Congrats <span class="citation">@AnaBetty2304</span> <span class="citation">@josschavezf1</span> <span class="citation">@cmvaldezcordova</span> for your presentations at #BioC2019 today! 🇲🇽 💪🏽</p>
<p>Soon near you BiocManager::install(#PulmonDB #regutools #histoneSig) 😁</p>
¡Felicidades Ana, Joselyn y César por sus presentaciones hoy en el congreso de <span class="citation">@Bioconductor</span>! 👏🏽 🙌🏽 <a href="https://t.co/sbv9u823sL" class="uri">https://t.co/sbv9u823sL</a>
</td>
<td style="text-align:right;">
0.9935634
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Love seeing in the talks already the strong sense of community in <span class="citation">@Bioconductor</span> software development. Lots of people credited and thanked, people working together to create great tools, and building upon the foundational work of the core team #BioC2019 #rstats
</td>
<td style="text-align:right;">
0.9932273
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<span class="citation">@Bioconductor</span> #bioc2019 in my (M. Morgan) thanks this AM I meant to include a special shout-out to Tom Carroll <span class="citation">@Rockefeller_BRC</span> and Kelly Ruggles <span class="citation">@kruggle</span> for amazing and heroic work as local organizers; the event has been exceptionally popular and successful due to their effort
</td>
<td style="text-align:right;">
0.9932273
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
Thank you, organizers, for an excellent #bioc2019. Beautiful location, great facilities, stimulating science, great talks, workshops and people. Especially, thanks to local hosts Thomas Carroll, Kelly Ruggles and <span class="citation">@LeviWaldron1</span>, Matthew McCall <a href="https://t.co/6gnh6oqOhX" class="uri">https://t.co/6gnh6oqOhX</a>
</td>
<td style="text-align:right;">
0.9932273
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
<span class="citation">@lievenClement</span> LC very clearly explaining the mass spec workflow and the challenges with proteomics quantitation and identification. Need to look up MS2 peaks in databases. “Proteomics people are addicted to big databases.” #BioC2019 <a href="https://t.co/rAXds6QWVD" class="uri">https://t.co/rAXds6QWVD</a>
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
robertclab
</td>
<td style="text-align:left;">
Today I’ll be giving a talk on technical aspects of this work at the <span class="citation">@Bioconductor</span> meeting #bioc2019 and on Sept 20th <span class="citation">@CostaCoto</span> will be giving a talk at the <span class="citation">@JENS_Congress</span> on its contribution to our understanding of the postnatal effects of the fetal inflammatory response 7/7
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
<tr>
<td style="text-align:left;">
k3yavi
</td>
<td style="text-align:left;">
#BioC2019 My first bioconductor conference and it couldn’t have been better. I met with so many brilliant researchers and exchanged ideas. I never realized how the idea of forming a community of similar mind researchers can be so powerful to move science forward, it’s incredible!
</td>
<td style="text-align:right;">
0.9928543
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">9</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table class="bioc-table">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Type
</th>
<th style="text-align:left;">
Link
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
BioconductorOnContainers
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/nturaga/bioconductoroncontainers" class="uri">https://github.com/nturaga/bioconductoroncontainers</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
BioCor
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/BioCor" class="uri">https://bioconductor.org/packages/BioCor</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
BiocStickers
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/bioconductor/biocstickers" class="uri">https://github.com/bioconductor/biocstickers</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
BiocWorkshops2019
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/bioconductor/biocworkshops2019/blob/master/readme.md" class="uri">https://github.com/bioconductor/biocworkshops2019/blob/master/readme.md</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
CellBench
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/shians/cellbench" class="uri">https://github.com/shians/cellbench</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
CellBench
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/CellBench" class="uri">https://bioconductor.org/packages/CellBench</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
DeepBlueR
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/DeepBlueR" class="uri">https://bioconductor.org/packages/DeepBlueR</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
fishpond
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/fishpond" class="uri">https://bioconductor.org/packages/fishpond</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
histonesig
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/semibah/histonesig" class="uri">https://github.com/semibah/histonesig</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
iSEE
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/csoneson/isee" class="uri">https://github.com/csoneson/isee</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
iSEE
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/iSEE" class="uri">https://bioconductor.org/packages/iSEE</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
kipoi
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/kipoi/kipoi" class="uri">https://github.com/kipoi/kipoi</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
mbkmeans
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/mbkmeans" class="uri">https://bioconductor.org/packages/mbkmeans</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
mogsa
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/mogsa" class="uri">https://bioconductor.org/packages/mogsa</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
MoonlightR
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/MoonlightR" class="uri">https://bioconductor.org/packages/MoonlightR</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
muscat
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/helenalc/muscat" class="uri">https://github.com/helenalc/muscat</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
PharmacoGx
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/PharmacoGx" class="uri">https://bioconductor.org/packages/PharmacoGx</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
Presentations
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/siminab/presentations/blob/master/bioc%202019/boca%20bioc%202019%20github.pdf" class="uri">https://github.com/siminab/presentations/blob/master/bioc%202019/boca%20bioc%202019%20github.pdf</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
PulmonDB
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/anabva/pulmondb" class="uri">https://github.com/anabva/pulmondb</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
regutools
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/josschavezf/regutools" class="uri">https://github.com/josschavezf/regutools</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
regutools
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/comunidadbioinfo/regutools" class="uri">https://github.com/comunidadbioinfo/regutools</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
SGSeq
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/SGSeq" class="uri">https://bioconductor.org/packages/SGSeq</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
snapr
</td>
<td style="text-align:left;">
GitHub
</td>
<td style="text-align:left;">
<a href="https://github.com/langmead-lab/snapr" class="uri">https://github.com/langmead-lab/snapr</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
TCGAbiolinks
</td>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<a href="https://bioconductor.org/packages/TCGAbiolinks" class="uri">https://bioconductor.org/packages/TCGAbiolinks</a>
</td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## R Under development (unstable) (2020-12-24 r79679)
## Platform: x86_64-pc-linux-gnu (64-bit)
## Running under: Ubuntu 20.04.1 LTS
## 
## Matrix products: default
## BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] stats     graphics  grDevices datasets  utils     methods   base     
## 
## other attached packages:
##  [1] fs_1.5.0            here_1.0.1          kableExtra_1.3.1   
##  [4] knitr_1.30          magick_2.5.2        webshot_0.5.2      
##  [7] viridis_0.5.1       viridisLite_0.3.0   wordcloud_2.6      
## [10] RColorBrewer_1.1-2  ggraph_2.0.4        ggrepel_0.8.2      
## [13] ggplot2_3.3.2       topicmodels_0.2-11  tidytext_0.2.6     
## [16] igraph_1.2.6        stringr_1.4.0       purrr_0.3.4        
## [19] forcats_0.5.0       lubridate_1.7.9.2   tidyr_1.1.2        
## [22] dplyr_1.0.2         rtweet_0.7.0        BiocManager_1.30.10
## 
## loaded via a namespace (and not attached):
##  [1] httr_1.4.2         tidygraph_1.2.0    jsonlite_1.7.2     assertthat_0.2.1  
##  [5] askpass_1.1        highr_0.8          stats4_4.1.0       renv_0.12.3       
##  [9] yaml_2.2.1         slam_0.1-48        pillar_1.4.7       lattice_0.20-41   
## [13] glue_1.4.2         digest_0.6.27      polyclip_1.10-0    rvest_0.3.6       
## [17] colorspace_2.0-0   plyr_1.8.6         htmltools_0.5.0    Matrix_1.2-18     
## [21] tm_0.7-8           pkgconfig_2.0.3    scales_1.1.1       processx_3.4.5    
## [25] tweenr_1.0.1       ggforce_0.3.2      tibble_3.0.4       openssl_1.4.3     
## [29] generics_0.1.0     farver_2.0.3       ellipsis_0.3.1     withr_2.3.0       
## [33] cli_2.2.0          NLP_0.2-1          magrittr_2.0.1     crayon_1.3.4      
## [37] ps_1.5.0           evaluate_0.14      tokenizers_0.2.1   janeaustenr_0.1.5 
## [41] fansi_0.4.1        SnowballC_0.7.0    MASS_7.3-53        xml2_1.3.2        
## [45] tools_4.1.0        lifecycle_0.2.0    munsell_0.5.0      callr_3.5.1       
## [49] compiler_4.1.0     rlang_0.4.9        grid_4.1.0         rstudioapi_0.13   
## [53] labeling_0.4.2     rmarkdown_2.6      gtable_0.3.0       curl_4.3          
## [57] reshape2_1.4.4     graphlayouts_0.7.1 R6_2.5.0           gridExtra_2.3     
## [61] utf8_1.1.4         rprojroot_2.0.2    modeltools_0.2-23  stringi_1.5.3     
## [65] parallel_4.1.0     Rcpp_1.0.5         vctrs_0.3.5        tidyselect_1.1.0  
## [69] xfun_0.19</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
